dev/master_sha.go

```diff
@@ -14,6 +14,17 @@ type build struct {
 }
 
 func main() {
+	// If we have a SHA override then use it.
+	if os.Getenv("SHA") != "" {
```

Comment:
nit: Maybe to avoid calling `Getenv` twice you can do something like:

```suggestion
	if sha := os.Getenv("SHA"); sha != "" {
```

Not a big deal though.

Reply:
I'm not really concerned with performance here, tbh. You are right that it's faster but... yeah, it doesn't really matter in this case. :)

Reply:
I wasn't worried about performance either, just thought it was more elegant. It's also technically possible that the value of the env var can change between the two calls, now that I think about it.

Reply:
That shouldn't be possible, really but on the other hand we don't lose anything by using your approach and it's also more idiomatic for Go, so sure, let's do it.